doc/classes/GeometryInstance.xml

```diff
@@ -14,32 +14,54 @@
 	</methods>
 	<members>
 		<member name="cast_shadow" type="int" setter="set_cast_shadows_setting" getter="get_cast_shadows_setting" enum="GeometryInstance.ShadowCastingSetting">
+			The currently selected shadow casting flag.
```

Comment:
Whenever possible, remove extra words that don't add content, ex: "currently".

When a member uses named constants, the standard has been to refer to them in the description, so the reader knows where to look. "See SHADOW_CASTING_SETTING_* constants for values."